src/main/scala/firrtl/Utils.scala

```diff
@@ -428,6 +437,14 @@ object Utils {
          case REVERSE => swap(d)
       }
    }
+   def times (g: Gender, d: Direction): Direction = times(d, g)
+   // TODO Does this make sense?
+   def times (d: Direction, g: Gender): Direction = g match {
+     case FEMALE => d
+     case MALE => swap(d) // MALE == INPUT == REVERSE
+     case _ => error(s"times called on invalid Gender: $g")
```

Comment:
Not a big deal, but since this is just an internal compile error that is completely unhelpful to the user, I'd just leave off the case _ entirely and let it throw a match error... unless this has frequently proved useful for debugging.